Formula/dockerize.rb

```diff
@@ -0,0 +1,23 @@
+class Dockerize < Formula
+  desc "Utility to simplify running applications in docker containers"
+  homepage "https://github.com/jwilder/dockerize"
+  url "https://github.com/jwilder/dockerize.git",
+      :tag      => "v0.6.1",
+      :revision => "7c5cd7c34dcf1c81f6b4db132ebceabdaae17153"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/jwilder/dockerize").install buildpath.children
+    ENV.append_path "PATH", buildpath/"bin"
+    cd "src/github.com/jwilder/dockerize" do
+      system "make", "dist"
+      bin.install "dist/darwin/amd64/dockerize"
+    end
+  end
+
+  test do
+    system "dist/darwin/amd64/dockerize", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
@mrmachine Can you add some tests? This recipe is really nice to have in homebrew

Reply:
What about 
```
dockerize -wait http://www.google.com -wait-retry-interval=1s -timeout 5s
```

Reply:
You don't have access to build time files, this should be `bin/"dockerize"`

Reply:
As @mrmachine doesn't respond anymore, I created a new PR to address this: https://github.com/Homebrew/homebrew-core/pull/37407.

Please close this PR.